### PR TITLE
Fixes for CI

### PR DESCRIPTION
**Pull request by tnozicka**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,20 +1 @@
-/_output
```
Comment (deads2k) [inline .gitignore]: pretty sure we want this and the IDE ones.

Comment (tnozicka) [reply .gitignore]: `_output` is not generated anymore

IDE ones don't belong there but to user's personal global .gitignore file

Comment (deads2k) [reply .gitignore]: > IDE ones don't belong there but to user's personal global .gitignore file

Perhaps purity-wise, but I don't see a reason to make life difficult.  Also, just saw that you updated this.  If it goes green then I'll close mine.

**Comment by deads2k** [comment]
removing WIP

/lgtm

Let's get lucky.

**Comment by deads2k** [comment]
@tnozicka well, I have a working verify now and a broken unit : https://github.com/openshift/cluster-openshift-apiserver-operator/pull/6

**Pull request merged**